warehouse/templates/pages/help.html

```diff
@@ -3,7 +3,7 @@
  # you may not use this file except in compliance with the License.
  # You may obtain a copy of the License at
  #
- # http://www.apache.org/licenses/LICENSE-2.0
+ # //www.apache.org/licenses/LICENSE-2.0
```

Comment:
hmmm... because I ran a replace all script over the file. 
I don't think the linter throws an error, because it is in a comment, which makes sense.  I'll revert the change :)